html/elements/link.json

```
@@ -108,7 +108,8 @@
                 "version_added": "79"
               },
               "firefox": {
-                "version_added": "18"
+                "version_added": "18",
+                "notes": "Prior to Firefox 83, crossorigin was not supported for rel=icon."
```

suggestion: nitpicky style stuff

```suggestion
                "notes": "Before Firefox 83, <code>crossorigin</code> is not supported for <code>rel=\"icon\"</code>."
```